src/Nox.Lib/Exceptions/EntityAttributeIsNotNullableException.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Nox.Exceptions;
+
+/// <summary>
+/// trying to set a null to a non nullable property
+/// </summary>
+public class EntityAttributeIsNotNullableException : Exception
```

Comment:
Can we delete this exception, I'm not seeing any usage. I know its not related with your PR, but lets improve this ? :)

Reply:
sure, it's removed now